scenarios/ocd/CreateAKSDeployment/README.md

```diff
@@ -37,6 +43,57 @@ Results:
 }
 ```
 
+## Create a virtual network and subnet
+
+A virtual network is the fundamental building block for private networks in Azure. Azure Virtual Network enables Azure resources like VMs to securely communicate with each other and the internet.
+
+```bash
+az network vnet create \
+    --resource-group $MY_RESOURCE_GROUP_NAME \
+    --location $MY_LOCATION \
+    --name $MY_VNET_NAME \
+    --address-prefix $MY_VNET_PREFIX \
+    --subnet-name $MY_SN_NAME \
+    --subnet-prefixes $MY_SN_PREFIX
+```
+Results:
+
+```expected_similarity=0.3
```

Comment:
expected_similarity needs to be declared as an html comment above the code block and code block should be annotated with `json`